### PR TITLE
linux: add PR_SET_PTRACER_ANY

### LINKED ISSUE #3085: Missing `PR_SET_PTRACER_ANY`
[`PR_SET_PTRACER`](https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/prctl.h#L155) is present (https://docs.rs/libc/latest/libc/constant.PR_SET_PTRACER.html) but [`PR_SET_PTRACER_ANY`](https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/prctl.h#L156) is missing (https://docs.rs/libc/latest/libc/constant.PR_SET_PTRACER.html?search=PR_SET_PTRACER_ANY).

### PR DESCRIPTION
**Pull request by howjmay**
fixed #3085

### CODE REVIEW

File: src/fuchsia/mod.rs [hunk 1]
```diff
@@ -2602,6 +2602,7 @@ pub const PR_SET_MM_MAP: ::c_int = 14;
 pub const PR_SET_MM_MAP_SIZE: ::c_int = 15;
 
 pub const PR_SET_PTRACER: ::c_int = 0x59616d61;
+pub const PR_SET_PTRACER_ANY: ::c_int = 0xffffffffffffffff;
```
Comment (devnexen) [inline src/fuchsia/mod.rs:2605]: The type is most likely wrong, too small to carry this value, ought to be `c_ulong`

Comment (howjmay) [reply src/fuchsia/mod.rs]: done

Comment (devnexen) [reply src/fuchsia/mod.rs]: cool, all tests pass it seems 🙂

**Comment by howjmay** [comment]
thanks

**Pull request merged**